app/code/Magento/Sales/Test/Mftf/Test/AdminOpenCreditmemoViewPageWithWrongCreditmemoIdTest.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminOpenCreditmemoViewPageWithWrongCreditmemoIdTest">
+        <annotations>
+            <stories value="Creditmemo Page With Wrong Creditmemo Id"/>
+            <title value="Open Creditmemo View Page with Wrong Creditmemo Id"/>
+            <description value="Open Creditmemo View Page with Wrong Creditmemo Id."/>
+            <severity value="MAJOR"/>
+            <group value="sales"/>
+        </annotations>
+
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="LoginAsAdmin"/>
+        </before>
+
+        <after>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+
+        <actionGroup ref="AdminGoToCreditmemoViewActionGroup" stepKey="navigateOpenCreditmemoViewPage">
+            <argument name="identifier" value="test"/>
+        </actionGroup>
+
+        <waitForPageLoad stepKey="waitForPageLoad"/>
+
+        <seeInCurrentUrl url="{{AdminCreditmemosGridPage.url}}" stepKey="redirectToCreditmemosGridPage"/>
+
+        <see selector="{{AdminMessagesSection.error}}" userInput='This creditmemo no longer exists.'
```

Comment:
Please, use the following action group instead

https://github.com/magento/magento2/blob/83e3487a12e1165dde0c9175fbad45b9bfe018c3/app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertMessageInAdminPanelActionGroup.xml#L11

Reply:
Hi @rogyar what is the need of changing action group name for this file as it belongs to Magento_Backend module, i have done commits on Sales and Shipping module.

Reply:
The main reason is following the best practices and building the test body using action groups only. If we already have an action group for some purpose it's recommended to use it rather than introduce a new check.